ds_judgements_public_ui/sass/includes/_header.scss

```diff
@@ -51,6 +51,18 @@
     }
   }
 
+  &__breadcrumbs-block {
+    margin-left: $space-4;
+    padding: $space-2;
+  }
+
+  &__beta {
```

Comment:
Is this _actually_ anything to do with the fact the service is beta? Could it be better named?